draft-ietf-quic-recovery.md

```diff
@@ -623,10 +623,21 @@ time.
 
 ### Sending Probe Packets
 
-When a PTO timer expires, the sender MUST send one ack-eliciting packet as a
-probe, unless there is nothing to send. A sender MAY send up to two
-ack-eliciting packets, to avoid an expensive consecutive PTO expiration due
-to a single packet loss.
+When a PTO timer expires, a sender MUST send at least one ack-eliciting packet
+as a probe, unless there is no data available to send.  An endpoint MAY send up
+to two ack-eliciting packets, to avoid an expensive consecutive PTO expiration
+due to a single packet loss.
+
+It is possible that the sender has no new or previously-sent data to send.  For
+example, this can happen when there is no new application data to send and all
+streams with data in flight are reset (see Section 13.2 of {{QUIC-TRANSPORT}}).
+Under these conditions, the sender SHOULD send PING or other ack-eliciting
+frames in up to two packets, re-arming the PTO timer.
+
+When there is no data to send, the sender MAY mark any packets still in flight
```

Comment:
I'd like to be clear that this is an alternative, so maybe say "Alternatively, when there is..."?